src/library-authoring/components/LibraryComponents.test.tsx

```diff
@@ -163,6 +164,17 @@ describe('<LibraryComponents />', () => {
     expect(screen.queryByRole('button', { name: /add component/i })).not.toBeInTheDocument();
   });
 
+  it('should render a spinner while loading', async () => {
+    mockUseSearchContext.mockReturnValue({
+      ...data,
+      isLoading: true,
+    });
+
+    render(<RootWrapper />);
+    const spinner = await screen.findByRole('status');
+    expect(spinner.textContent).toEqual('Loading...');
```

Comment:
I'm not sure why, but this fixes it:
```suggestion
    const spinner = await screen.findByRole('status', { hidden: true });
    expect(spinner.textContent).toEqual('Loading...');
```
Maybe the spinner starts hidden because of the animation.. But why does this work in other tests?

This also works, which I think is less hacky:
```suggestion
    expect(screen.getByText('Loading...')).toBeInTheDocument();
```

Reply:
Oh wow thank you for figuring this out! I couldn't crack it.

Reply:
Fixed with [add30fb](https://github.com/openedx/frontend-app-authoring/pull/1331/commits/add30fbcb45a487878c141e16d50c737d4831efe)